pkg/cli-runtime/testing/command_table.go

```diff
@@ -304,3 +343,60 @@ func fakeExecCommand(helper string) func(context.Context, string, ...string) *ex
 		return cmd
 	}
 }
+
+func trimTrailingSpaces(in string) string {
+	lines := strings.Split(in, "\n")
+	for i := len(lines) - 1; i >= 0; i-- {
+		lines[i] = strings.TrimRight(lines[i], " ")
+	}
+	return strings.Join(lines, "\n")
```

Comment:
declare line break constant in separate lib and use that across all files. When the time comes to add support for line breaks across diff platforms then we can make that change easily.

Reply:
test constants used instead